README.md

```diff
@@ -12,7 +12,13 @@ It doesn't have any external dependecy except bootstrap-datepicker.
 
 ## Installation
 
-If you are using Ember CLI 0.1.5 and higher, just run within your project directory:
+If you are using Ember CLI 0.2.3 or higher, just run within your project directory:
+
+```bash
+ember install:addon ember-cli-bootstrap-datepicker
```

Comment:
There might be a copy-paste mistake. I guess the command should be `ember install ember-cli-bootstrap-datepicker` (`:addon` prefix is omitted), otherwise there's no difference with installation instructions for Ember CLI less than 0.2.3 :wink: